### ISSUE TITLE
Listmonk 4.1.0 Campaign Editor Has Cursor Issue

### ISSUE THREAD

**Issue by PackMeister** [knadh/listmonk#2194]
- listmonk: 4.1.0
 - OS: OSX 15.1.1
 - Browser: Brave 1.73.97

After successfully cloning a previous campaign so I could update particulars for a new campaign I noticed the editor's cursor didn't reflect the actual location where new text was being added or old text deleted, but rather two lines above.

Very strange and annoying.

**Comment by abnogz** [comment]
Hi! @PackMeister , I have a similar issue, and it seems to happen when a line in the code is too large to fit in the editor, and from that line onwards the cursor behaves like it is one or two lines above where it is located.

**Comment by mevinbabuc** [comment]
Yup, this exists. If you do a commdand+a in the editor, you can see text in white.

**Comment by MaximilianKohler** [comment]
Not stale.

**Comment by knadh** [comment]
This issue was fixed in v5.0.0 where the HTML editor component was fully replaced with a much more robust lib.
